doc/adr/20190318-deploy-to-gsp.md

```diff
@@ -0,0 +1,26 @@
+# Deploy proxy node on GDS Supported Platform
+
+Date: 2019-03-18
+
+## Status
+
+Accepted
+
+## Context
+
+GDS requires that all GDS-run services are hosted and maintained internally and in a consistent manner. For microservice architectures such as the proxy node, the technology GDS has chosen is Kubernetes deployed to Amazon AWS EC2 instances, dubbed the GDS Supported Platform (GSP). Applications running on the platform are containerised using Docker.
```

Comment:
```suggestion
GDS requires that all GDS-run services are hosted and maintained internally and in a consistent manner. For microservice architectures such as the proxy node, the technology GDS has chosen is Kubernetes deployed to AWS instances, dubbed the GDS Supported Platform (GSP). Applications running on the platform are containerised using Docker.
```

This is going away soon. I would refrain from declaring specifics for that matter.